android/build.gradle

```diff
@@ -2,18 +2,18 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "30.0.2"
+        buildToolsVersion = "33.0.0"
         minSdkVersion = 21
-        compileSdkVersion = 30
-        targetSdkVersion = 30
+        compileSdkVersion = 33
+        targetSdkVersion = 33
         ndkVersion = "21.4.7075529"
```

Comment:
This is back to my previous question on ndkVersions

Reply:
Same as in the other response. NDK affects C/C++ native code. It should be irrelevant here. I updated it while testing and nothing changed that I could notice.